validate.go

```diff
@@ -235,6 +236,10 @@ func validateFormat(path *PathBuffer, str string, s *Schema, res *ValidateResult
 		if ip := net.ParseIP(str); ip == nil || ip.To16() == nil {
 			res.Add(path, str, validation.MsgExpectedRFC2373IPv6)
 		}
+	case "ip":
+		if _, err := netip.ParseAddr(str); err != nil {
+			res.Add(path, str, validation.MsgExpectedRFCIPAddr)
+		}
```

Comment:
Love this! One concern I have is that `ipv4` and `ipv6` are actually in the JSON Schema spec but `ip` is not. See https://json-schema.org/understanding-json-schema/reference/type#ip-addresses.

I wonder if we should support all three? The two standard ones, and then `ip` for either one since Go supports it well. What do you think?